src/Simulator/IncrementalRedstoneSimulator/RedstoneSimulatorChunkData.h

```diff
@@ -93,6 +93,7 @@ class cIncrementalRedstoneSimulatorChunkData final : public cRedstoneSimulatorCh
 	/** Structure storing position of mechanism + it's delay ticks (countdown) & if to power on. */
 	std::unordered_map<Vector3i, std::pair<int, bool>, VectorHasher<int>> m_MechanismDelays;
 
+	std::unordered_map<Vector3i, cTickTimeLong> PistonExtensionWorldTickTime;
```

Comment:
```suggestion
	std::unordered_map<Vector3i, cTickTimeLong, VectorHasher<int>> PistonExtensionWorldTickTime;
```
Need the hasher to fix compile.